### ISSUE TITLE
`npm install` errors out after running `npx svelte-migrate sveltekit-2`

### ISSUE THREAD

**Issue by nmalchy** [sveltejs/kit#11351]
### Describe the bug

Running `npm install` does not work after running the migration command to sveltekit v2

Here is the error I am getting:
![image](https://github.com/sveltejs/kit/assets/50896885/00841585-9e91-41f2-8027-86381aab24da)

Here is my diff after the migration command:
![image](https://github.com/sveltejs/kit/assets/50896885/99fcad06-772e-4ba1-8845-b27dda58b6c6)

### Reproduction

Run `npx svelte-migrate sveltekit-2` and then `npm install`

### System Info

```Shell
Windows 10, git bash
```

### Severity

blocking an upgrade

**Comment by hanszoons** [comment]
Yes i had to install the new versions manually again too to get rid of that error.

**Comment by nmalchy** [comment]
> Yes i had to install the new versions manually again too to get rid of that error.

Tried that and still not working, I think it has to do with my other dependencies like Sentry, I'm sure a fresh project will be fine but I think I just have to wait for the other projects to catch up, it has only been a day.

**Comment by peterpeterparker** [comment]
```
npm rm @sveltejs/adapter-static @sveltejs/kit @sveltejs/vite-plugin-svelte
npm i
npm i @sveltejs/adapter-static @sveltejs/kit @sveltejs/vite-plugin-svelte -D
```

**Comment by SKAT-16** [comment]
> ```
> npm rm @sveltejs/adapter-static @sveltejs/kit @sveltejs/vite-plugin-svelte
> npm i
> npm i @sveltejs/adapter-static @sveltejs/kit @sveltejs/vite-plugin-svelte -D
> ```

this hasn't worked for me

**Comment by SKAT-16** [comment]
This worked for me
I used --force flag based on the answer from @peterpeterparker
npm rm @sveltejs/kit @sveltejs/vite-plugin-svelte --force
npm i
npm i @sveltejs/adapter-static @sveltejs/kit @sveltejs/vite-plugin-svelte -D

Hope this helps someone else.

**Comment by vytenisstaugaitis** [comment]
`rm package-lock.json
npm install`
